### PR TITLE
[SeaTunnel #910] Support spark-hive sink

### LINKED ISSUE #910: [Feature][Plugin] Support spark-hive sink
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

Support spark-hive sink

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by simon824**
## Purpose of this pull request
closes #910

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] Change does not need document change, or I will submit document change to https://github.com/apache/incubator-seatunnel-website later

### CODE REVIEW

File: docs/en/configuration/sink-plugins/Hive.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+### Description
+
+Write Rows to [Apache Hive](https://hive.apache.org).
+
+### Options
+
+| name                                    | type          | required | default value |
+| --------------------------------------- | ------------- | -------- | ------------- |
+| [sql](#hql)                             | string        | no       | -             |
```
Comment (RickyHuo) [inline docs/en/configuration/sink-plugins/Hive.md:11]: #sql

**Review by RickyHuo — CHANGES_REQUESTED** [review]
In the `Hive.md-Options`, the redirect of param name is invalid. Please check it.

You should write like this

> | [save_mode](#save_mode-string)      | string        | no       | -             |

File: docs/en/configuration/sink-plugins/Hive.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+| [source_table_name](#source_table_name) | string        | no       | -             |
+| [result_table_name](#result_table_name) | string        | no       | -             |
+| [sink_columns](#sink_columns)           | string        | no       | -             |
+| [save_mode](#save_mode)                 | string        | no       | -             |
+| [partition_by](#partition_by)           | Array[string] | no       | -             |
+
+##### sql[string]
```
Comment (RickyHuo) [inline docs/en/configuration/sink-plugins/Hive.md:18]: sql [string]

**Comment by simon824** [comment]
> In the `Hive.md-Options`, the redirect of param name is invalid. Please check it.
> 
> You should write like this
> 
> ```
>  | [save_mode](#save_mode-string)      | string        | no       | -             |
> ```

Updated, thanks for your review.

**Review by RickyHuo — APPROVED** [review]
LGTM

**Pull request merged**